arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -86,28 +100,47 @@
                             value: facetFilterText, 
                             valueUpdate: 'keyup'
                         "
+                        aria-label="{% trans 'Filter search facets' %}"
```

Comment:
we shouldn't be using {% trans %} tags anymore.  We should be using js tanslation objects.